collect_app/src/main/java/org/odk/collect/android/database/ActivityLogger.java

```diff
@@ -115,7 +120,27 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
     public ActivityLogger(String deviceId) {
         this.deviceId = deviceId;
         loggingEnabled = new File(Collect.LOG_PATH, ENABLE_LOGGING).exists();
-        open();
+
+        if (loggingEnabled) {
+            open();
+
+            if (isFirstTime()) {
+                sendAnalyticsEvent();
+                GeneralSharedPreferences.getInstance().save(KEY_ACTIVITY_LOGGER_EVENT, false);
+            }
+        }
+    }
+
+    private void sendAnalyticsEvent() {
+        Collect.getInstance()
+                .getDefaultTracker()
+                .send(new HitBuilders.EventBuilder()
+                        .setLabel("ActivityLogger is enabled")
```

Comment:
It may be worth considering adding a Category & Action. Labels are used for additional information.

See: [About Events](https://support.google.com/analytics/answer/1033068)

Reply:
Agreed. How about:
* Category: `ActivityLogger`
* Action: `Enabled`